website_compassion/forms/fund_donation_form.py

```diff
@@ -0,0 +1,107 @@
+##############################################################################
+#
+#    Copyright (C) 2021 Compassion CH (http://www.compassion.ch)
+#    @author: Jonathan Guerne <guernej@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo import models, fields, _
+
+
+class FundDonationForm(models.AbstractModel):
+    _name = "cms.form.fund.donation"
+    _inherit = ["cms.form.payment"]
+
+    _display_type = "full"
+    _form_model = "res.partner"
+
+    amount = fields.Float(required=True)
+    fund = fields.Many2one("product.product")
+
+    _form_model_fields = [
+        "amount",
+    ]
```

Comment:
This is wrong, it would mean the field "amount" is in the underlying model which here you set to "res.partner".
```suggestion
```